packages/maskbook/src/components/InjectedComponents/BreakdownDialog.tsx

```diff
@@ -0,0 +1,193 @@
+} from '@material-ui/core'
+import { Info as InfoIcon } from '@material-ui/icons'
+import { InjectedDialog } from '../shared/InjectedDialog'
+import { makeStyles } from '@material-ui/core/styles'
+import { useStylesExtends } from '../custom-ui-helper'
+import { MaskbookIcon } from '../../resources/MaskbookIcon'
+import { AirdropIcon } from '../../resources/AirdropIcon'
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        dialogPaper: {
+            background: 'linear-gradient(180.43deg, #04277B 26.69%, #6B94F2 99.57%)',
+        },
+        content: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            padding: theme.spacing(6.25, 4.375),
+            color: '#fff',
+        },
+        dialogTitle: {
+            backgroundColor: '#04277B',
+            color: '#fff',
+            borderBottom: 'none !important',
+        },
+        logo: {
+            width: 96,
+            height: 96,
+        },
+        amount: {
+            fontSize: 32,
+            marginTop: theme.spacing(5),
+        },
+        balance: {
+            display: 'flex',
+            justifyContent: 'space-between',
+            fontSize: 14,
+            width: '100%',
+            margin: theme.spacing(3.75, 0, 2.5),
+        },
+        airDropContainer: {
+            background: 'linear-gradient(90deg, #2174F6 0%, #00C6FB 100%)',
+            borderRadius: 10,
+            width: '100%',
+        },
+        airdropContent: {
+            borderBottom: '1px dashed rgba(255,255,255, 0.5)',
+            padding: theme.spacing(2.5),
+            display: 'flex',
+            justifyContent: 'space-between',
+            color: '#fff',
+            fontSize: 14,
+            position: 'relative',
+        },
+        checkAddress: {
+            padding: theme.spacing(2.5),
+            fontSize: 13,
+            color: '#fff',
+        },
+        airDropIcon: {
+            width: 70,
+            height: 79,
+            position: 'absolute',
+            left: '17%',
+            top: 5,
+        },
+        checkAddressInput: {
+            flex: 1,
+            height: 48,
+            color: '#fff',
+            '& > fieldset': {
+                borderColor: '#F3F3F4',
+            },
+        },
+        ITOContainer: {
+            borderRadius: 10,
+            width: '100%',
+            background: 'linear-gradient(90deg, #FE686F 0%, #F78CA0 100%);',
+            marginTop: theme.spacing(2.5),
+        },
+        ITOContent: {
+            display: 'flex',
+            justifyContent: 'space-between',
+            padding: theme.spacing(2.5),
+        },
+        ITOAlertContainer: {
+            padding: theme.spacing(0, 2.5, 2.5, 2.5),
+        },
+        ITOAlert: {
+            backgroundColor: 'rgba(255,255,255,0.2)',
+            color: '#fff',
+        },
+        button: {
+            background: 'rgba(255,255,255,.2)',
+            '&.Mui-disabled': {
```

Comment:
Please don't use this.

Reply:
Here  I can't use `<Button classes={{ disabled: disabledClass }} ... />,` what is the right way to do it? 
use `"&[disabled]"`?

Reply:
why you can't use it?

Reply:
Because it doesn't replace .Mui-disabled to my class